pkg/controller/handlers.go

```diff
@@ -183,19 +186,18 @@ func (h Handler) PoliciesForApp(req router.Request, resp router.Response) error
 // proxy traffic to it). This doesn't block incoming network traffic from a different Acorn project in the same
 // cluster, but Acorn's built-in NetworkPolicies already take care of that.
 func PoliciesForIngress(req router.Request, resp router.Response) error {
-	// check if this is being called as a finalizer for a deleted Ingress
-	// we need this because we sometimes create policies in different namespaces from where their owning Ingresses live
-	if !req.Object.GetDeletionTimestamp().IsZero() {
+	ingress := req.Object.(*netv1.Ingress)
+
+	// Don't process the Ingress resource created for Acorn DNS, since it doesn't refer to any pods
+	if ingress.Name == systemIngress && ingress.Namespace == systemNamespace {
```

Comment:
Could we pull the `systemIngress` and `systemNamespace` from the runtime `system` package instead?

Reply:
Ah, I see. Not blocking so we can get this in whenever 👍🏻